waiter/src/waiter/service_description.clj

```diff
@@ -443,7 +443,9 @@
   [kv-store ^String token error-on-missing include-deleted]
   (let [{:strs [deleted run-as-user] :as token-data} (when token (kv/fetch kv-store token))
         token-data (when token-data ; populate token owner for backwards compatibility
-                     (update-in token-data ["owner"] (fn [current-owner] (or current-owner run-as-user))))]
+                     (cond-> token-data
```

Comment:
Nitpick:
```clojure
(-> token-data
   (update "owner" (fn [owner] (or owner run-as-user)))
   (update "previous" (fn [previous] (or previous {}))))
```

Reply:
I wanted to avoid creating and invoking that function. `update` also carries the connotation that we will be modifying the value using the existing value mapped against a key, which is not true in this case.